fiftyone/server/samples.py

```diff
@@ -44,7 +44,7 @@ class ImageSample(Sample):
 
 @gql.type
 class PointCloudSample(Sample):
-    pass
+    aspect_ratio: float
```

Comment:
`aspect_ratio` could be moved to the `Sample` interface now that all three types require it